src/nvim/help.c

```diff
@@ -653,35 +652,6 @@ void fix_help_buffer(void)
     curbuf->b_ro_locked--;
   }
```

Comment:
What does this part even do? According to my tests, help buffers are detected without it just fine (and doesn't this code only run on help buffers, so filetype should already be set)?

Reply:
Is this because Vim (unlike Neovim) may not run filetype by default?